tpv36_37/parameters.par

```diff
@@ -0,0 +1,107 @@
+&equations
+!yaml file defining spatial dependence of material properties
+MaterialFileName = 'material.yaml'
+!enable off-fault plasticity (ignored in Plasticity=0)
+Plasticity = 0
+!Tv = 0.03
+! Requires SeisSol v1.2.0 or newer
+numflux = 'rusanov' ! The numerical flux near the fault. Values: 'godunov', 'rusanov'
+numfluxnearfault = 'rusanov' ! The numerical flux near the fault (on cells which have at least one DR face). Values: 'godunov', 'rusanov'
+/
+
+&IniCondition
+/
+
+&DynamicRupture
+! linear slip-weakening friction
+FL = 16                                        ! Friction law  
+! See https://github.com/SeisSol/SeisSol/issues/546
+!yaml file defining spatial dependence of fault properties
+! Choose tpv36 or tpv37 yaml file (different cohesion)
+ModelFileName = 'fault_tpv36.yaml' !'fault_tpv37.yaml'
+
+!characteristic time of the forced nucleation
+t_0 = 0.5
+
+!reference vector for defining strike and dip direction
+XRef = 0.0
+YRef = -1.0
+ZRef = 0.0
+refPointMethod = 1
+
+OutputPointType = 5                            ! Type (0: no output, 3: ascii file, 4: paraview file, 5: 3+4)
+SlipRateOutputType=1        ! 0: (smoother) slip rate output evaluated from the difference between the velocity on both side of the fault
+                            ! 1: slip rate output evaluated from the fault tractions and the failure criterion (less smooth but usually more accurate where the rupture front is well developped)
+/
+
+!see: https://seissol.readthedocs.io/en/latest/fault-output.html
+!parameterize paraview file output
+&Elementwise
+printtimeinterval_sec = 1.0                    ! Time interval at which output will be written
+OutputMask = 1 1 1 1 1 1 1 1 1 1 1             ! turn on and off fault outputs
+refinement_strategy = 2
+refinement = 1
+/
+
+! parameterize ascii fault file outputs
+&Pickpoint
+printtimeinterval = 1       ! Index of printed info at timesteps
+OutputMask = 1 1 1 1 1 1 1 1 1 1 1 1        ! turn on and off fault outputs
+PPFileName = 'onfault_receivers.dat'
+/
+
+&SourceType
+/
+
+&SpongeLayer
```

Comment:
These sections should in principle be obsolete by now

Reply:
Thanks, deleted.